index.html

```diff
@@ -11,26 +11,27 @@
       href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
-    <title>Portfolio</title>
+    <link rel="shortcut icon" href="images/logo.png" />
+    <title>Rehema's Portfolio</title>
   </head>
   <body>
     <div class="page-content">
       <header class="header">
         <nav class="nav">
```

Comment:
Your navbar is showing the hamburger icon on bigger screens. It should only be visible on the smaller screens and not on the bigger screens.
Here is a reference screenshot of your app.
![image](https://user-images.githubusercontent.com/33268927/173548636-8ae995fa-505c-488c-a9bb-5a1e79794462.png)

- Optional. 
Your header layout is not matching the Figma template of your choosing. Kindly fix it.

Reply:
I don't get why this is what you are seeing from your side, the app looks okay from my side, I have tested it on both chrome and edge and this is how it looks. 

![image](https://user-images.githubusercontent.com/69511010/173597470-f11b7a8f-ef76-4c4e-951f-ded44c69ec16.png)
![image](https://user-images.githubusercontent.com/69511010/173597662-375a0dcc-d85a-4211-a3b5-96134e292ae8.png)
![image](https://user-images.githubusercontent.com/69511010/173597759-f73a8fd9-f476-4ab6-94e8-c2c421063690.png)
![image](https://user-images.githubusercontent.com/69511010/173597856-23091a54-63e9-4dde-a268-e3c9cf99cf8b.png)
![image](https://user-images.githubusercontent.com/69511010/173598602-47774164-3ce4-40cb-b35d-d758a21de164.png)
![image](https://user-images.githubusercontent.com/69511010/173599011-308d7b49-416a-4de0-af25-f5d55ee88084.png)